src/Common/AssemblyLoadingUtils.cs

```diff
@@ -185,7 +186,7 @@ private static void LoadAssembliesInDir(IHostEnvironment env, string dir, bool f
                 return;
 
             // Load all dlls in the given directory.
-            var paths = Directory.EnumerateFiles(dir, "*.dll");
+            var paths = Directory.EnumerateFiles(dir, "*.dll").Concat(Directory.EnumerateFiles(dir, "*.exe"));
```

Comment:
>Directory.EnumerateFiles(dir, "*.exe") [](start = 70, length = 38)

What exe file do you have in mind?
Can we instead move all classes to library and use "exe" only as startup thing?

Reply:
in any case, the comment above should also change

---
In reply to: [227578329](https://github.com/dotnet/machinelearning/pull/1350#discussion_r227578329) [](ancestors = 227578329)

Reply:
There is an exe file that contains some of the commands. I'll try moving them out to a separate library.

---
In reply to: [227578329](https://github.com/dotnet/machinelearning/pull/1350#discussion_r227578329) [](ancestors = 227578329)